src/components/Timer/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import './index.scss';
+function timer() {
+  return (
+    <div className="timer">
+      soy el timer <span className="timer__wi">WIIIIIIIIIIIIIIIIIIII</span>!!!!!!
+    </div>
+  );
+}
+
```

Comment:
Mismo comentario aplica aca tambien.
```suggestion
const timer = () => (
  <div className="timer">
    soy el timer <span className="timer__wi">WIIIIIIIIIIIIIIIIIIII</span>!!!!!!
  </div>
);


```